src/redux/books/books.js

```diff
@@ -0,0 +1,33 @@
+// Actions
+const Addbook = 'ADD_BOOK';
+const Removebook = 'REMOVE_BOOK';
+
```

Comment:
- [ ] It would be nice if the value of each action type needs to follow the [ducks pattern](https://github.com/erikras/ducks-modular-redux) :blush: which is a requirement of this project, so it will be something like this :arrow_down: 
```javascript
const ADD_BOOK = 'bookstore/books/ADD_BOOK; 
const REMOVE_BOOK = 'bookstore/books/REMOVE_BOOK'
```